test/lib/runners/baseRunner.js

```diff
@@ -0,0 +1,105 @@
+const
+  nexpect = require('nexpect'),
+  fs = require('fs'),
```

Comment:
fs, path, indentString and sanitize are no longer used here, we can remove their require